### PR TITLE
docs: Update changelog for release 2.2.0

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.1.2 - Jul 12 2021
+
+Incremental version for Teams Toolkit with following updates：
+
+Added:
+- UI Run button for debug and preview
+- Provide 'Report Issue' button in the pop-up error message to log issues when npm install fails
```
Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:9]: npm install fails or system error occurs

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+On-going A/B testing:
```
Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:11]: I think A/B testing is also a added feature? should we include it in the "Added" section?

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.1.2 - Jul 12 2021
```
Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:3]: The version is 2.2.0

**Comment by wenytang-ms** [comment]
please modify package.json version number in this pr

**Comment by qinezh** [comment]
> please modify package.json version number in this pr

@wenytang-ms updated, please help to review.

**Review by wenytang-ms — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Pull request merged**